src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1848,12 +1848,17 @@ public bool ExtensionsCollapsed
         {
             get
             {
-                // Threshold that determines if button should be displayed
-                if (RightExtensionsViewColumn.Width.Value < 20)
-                { extensionsCollapsed = true; }
-
+                // Special case: when the extension bar was never resized its size will be 2.
```

Comment:
@mmisol Can we hold on on this change, I feel this assumption may not be true in the future, after we implement https://jira.autodesk.com/browse/DYN-3144

Reply:
@QilongTang I don't see why. The work here is done and it leaves the code in a better state than what it was. The fact we may need to change it later does not fix the bug. In any case, if you think it's necessary, you may want to link this PR as reference for future work.

Reply:
@mmisol Let me ask in a different way. If `RightExtensionsViewColumn` was initialized with a different width, will this check still be valid？I was simply hoping we could fix two issues in a single shot

Reply:
@QilongTang The check would still be needed. Depending on how you do it, you may have to change the values here or not.